crates/bevy_pbr/src/meshlet/cull_clusters.wgsl

```diff
@@ -82,19 +83,15 @@ fn cull_clusters(
     let occlusion_culling_bounding_sphere_center_view_space = (view.view_from_world * vec4(occlusion_culling_bounding_sphere_center.xyz, 1.0)).xyz;
 #endif
 
-    var aabb = project_view_space_sphere_to_screen_space_aabb(occlusion_culling_bounding_sphere_center_view_space, occlusion_culling_bounding_sphere_radius);
```

Comment:
Please do not change cull_clusters.wgsl. I am modifying the occlusion culling soon to use AABBs instead.

Feel free to copy the code to use in other shaders, but leave this here.

Reply:
I don't agree with having duplicate code, but OK.